share/wake/lib/core/double.wake

```diff
@@ -13,65 +13,67 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 package wake
 
 # The Double type is an IEEE 754 64-bit precision number.
 # Any wake literal which include a decimal point or an exponent is parsed into a Double.
 
+
 # Unary absolute value operator for a Double.
 # dabs (+. 2.5) = 2.5
 # dabs (-. 2.5) = 2.5
-export def dabs (x: Double): Double =
-    (\_ prim "dabs") x
+export def dabs (x: Double): Double = (\_ prim "dabs") x
+
 
 # Unary negative sign for a Double.
 # -. (-. 2.5) =  2.5
 # -. (+. 2.5) = -2.5
-export def -. (x: Double): Double =
-    (\_ prim "dneg") x
+export def -. (x: Double): Double = (\_ prim "dneg") x
+
 
 # Unary positive sign for a Double.
 # +. 2.5 = 2.5
-export def +. (x: Double): Double =
-    x
+export def +. (x: Double): Double = x
+
 
 # Binary addition operator for Double values.
 # 1.1 +. 2.0 = 3.1
 # 0.1 +. 0.5 = 0.6
-export def (x: Double) +. (y: Double): Double =
-    (\_\_ prim "dadd") x y
+export def (x: Double) +. (y: Double): Double = (\_\_ prim "dadd") x y
+
 
 # Binary subtraction operator for Double values.
 # 4.0 -. 2.2 = 1.8
 # 1.1 -. 2.0 = -. 0.9
-export def (x: Double) -. (y: Double): Double =
-    (\_\_ prim "dsub") x y
+export def (x: Double) -. (y: Double): Double = (\_\_ prim "dsub") x y
+
 
 # Binary multiplication operator for Double values.
 # 2.0 *. 3.3 = 6.6
 # 2.0 *. 4.1 = 8.2
-export def (x: Double) *. (y: Double): Double =
-    (\_\_ prim "dmul") x y
+export def (x: Double) *. (y: Double): Double = (\_\_ prim "dmul") x y
+
 
 # Binary division operator for Double valuess.
 # 4.0 /. 2.0 = 2.0
 # 5.0 /. 2.0 = 2.5
-export def (x: Double) /. (y: Double): Double =
-    (\_\_ prim "ddiv") x y
+export def (x: Double) /. (y: Double): Double = (\_\_ prim "ddiv") x y
+
 
 # Binary exponentiation operator for Double values.
 # 2.0 ^. 3.0 = 8.0
 # 0.5 ^. 2.0 = 0.25
-export def (x: Double) ^. (y: Double): Double =
-    (\_\_ prim "dpow") x y
+export def (x: Double) ^. (y: Double): Double = (\_\_ prim "dpow") x y
+
 
 # Computes x*y + z with rounding only at the end.
 # The fused-multiply-add operation is useful in numeric algorithms.
 # It is necessary in order compute values with full precision.
 # dfma 2.0 3.0 1.0 = 7.0
 # dfma 1.0 1.0 1.0 = 3.0
-export def dfma (x: Double) (y: Double) (z: Double): Double =
-    (\_\_\_ prim "dfma") x y z
+export def dfma (x: Double) (y: Double) (z: Double): Double = (\_\_\_ prim "dfma") x y z
```

Comment:
I haven't yet written any lambda formatting rules, right now it is just spitting out the input unedited. This would be a good place to make your case on what the rule _should_ be :)

Reply:
Though I am working on the lambda rule right not to fix the internal repo, so you should post your suggestions soon :)

Reply:
Apologies for duplicating this commentary, but I made a similar observation elsewhere. I definitely prefer `\x \y \z` but am somewhat ambivalent what's chosen for `\_\_\_` since it's a matter of consistency (space) vs conciseness (no space) in that case.